signatures/suspicious_function_call.py

```diff
@@ -22,4 +22,4 @@ def process_output(self, output):
 
         if len(results) > 0:
             for result in results:
-                self.marks.add(f"{result} is evaluated using a suspicious pattern")
+                self.marks.append(f"{result} is evaluated using a suspicious pattern")

```

Comment:
You don't have the if for that one, is this one different and can't have duplicate, or is ok to have duplicate?

Reply:
Good catch